sdk/search/azure-search-documents/README.md

```diff
@@ -180,6 +432,28 @@ When you interact with Azure Cognitive Search using this Java client library, er
 to the same HTTP status codes returned for [REST API][rest_api] requests. For example, if you try to retrieve a document 
 that doesn't exist in your index, a `404` error is returned, indicating `Not Found`.
 
+### Handling Search Error Response
```

Comment:
Should this be a subsection of `### Troubleshooting`

Reply:
It is already in a subsection under `TroubleShooting`
Do you mean is it better to move to `Examples`?
This is a general case over SDKs. Customer might have experience on how to catch the problem, so we don't need to make it so obvious.

Reply:
If it is already in Troubleshooting then we are good!